innsender/src/main/kotlin/no/nav/soknad/innsending/util/mapping/tilleggsstonad/TilleggsstonadJson2JsonConverter.kt

```diff
@@ -0,0 +1,449 @@
+        ordnerDuFlyttingenSelvEllerKommerDuTilABrukeFlyttebyra = tilleggsstonad.ordnerDuFlyttingenSelvEllerKommerDuTilABrukeFlyttebyra,
+        jegFlytterSelv = tilleggsstonad.jegFlytterSelv,
+        jegVilBrukeFlyttebyra = tilleggsstonad.jegVilBrukeFlyttebyra,
+        jegHarInnhentetTilbudFraMinstToFlyttebyraerMenVelgerAFlytteSelv = tilleggsstonad.jegHarInnhentetTilbudFraMinstToFlyttebyraerMenVelgerAFlytteSelv
+    )
+}
+
+private fun convertToJsonBostotte(tilleggsstonad: Application): JsonBostottesoknad? {
+    if (tilleggsstonad.hvilkeAdresserHarDuBoutgifterPa == null
+        || tilleggsstonad.hvilkeBoutgifterSokerDuOmAFaDekket == null
+        || tilleggsstonad.startdatoDdMmAaaa == null
+        || tilleggsstonad.sluttdatoDdMmAaaa == null
+    ) return null
+
+    return JsonBostottesoknad(
+        aktivitetsperiode = JsonPeriode(tilleggsstonad.startdatoDdMmAaaa, tilleggsstonad.sluttdatoDdMmAaaa),
+        hvilkeBoutgifterSokerDuOmAFaDekket = tilleggsstonad.hvilkeBoutgifterSokerDuOmAFaDekket,
+        bostotteIForbindelseMedSamling = tilleggsstonad.bostotteIForbindelseMedSamling,
+        mottarDuBostotteFraKommunen = tilleggsstonad.mottarDuBostotteFraKommunen ?: "Nei", // "Ja" | "Nei"
+        bostottebelop = tilleggsstonad.hvorMyeBostotteMottarDu,
+        hvilkeAdresserHarDuBoutgifterPa = tilleggsstonad.hvilkeAdresserHarDuBoutgifterPa,
+        boutgifterPaHjemstedetMitt = tilleggsstonad.boutgifterPaHjemstedetMitt,
+        boutgifterPaAktivitetsadressen = tilleggsstonad.boutgifterPaAktivitetsadressen,
+        boutgifterJegHarHattPaHjemstedetMittMenSomHarOpphortIForbindelseMedAktiviteten = tilleggsstonad.boutgifterJegHarHattPaHjemstedetMittMenSomHarOpphortIForbindelseMedAktiviteten,
+        erDetMedisinskeForholdSomPavirkerUtgifteneDinePaAktivitetsstedet = tilleggsstonad.erDetMedisinskeForholdSomPavirkerUtgifteneDinePaAktivitetsstedet
+    )
+}
+
+private fun convertToLaeremiddelutgifter(tilleggsstonad: Application): JsonLaeremiddelutgifter? {
+    if (tilleggsstonad.hvilkenTypeUtdanningEllerOpplaeringSkalDuGjennomfore == null || tilleggsstonad.startdatoDdMmAaaa == null || tilleggsstonad.sluttdatoDdMmAaaa == null) return null
+
+    return JsonLaeremiddelutgifter(
+        aktivitetsperiode = JsonPeriode(
+            tilleggsstonad.startdatoDdMmAaaa,
+            tilleggsstonad.sluttdatoDdMmAaaa
+        ),
+        hvilkenTypeUtdanningEllerOpplaeringSkalDuGjennomfore = tilleggsstonad.hvilkenTypeUtdanningEllerOpplaeringSkalDuGjennomfore,
+        hvilketKursEllerAnnenFormForUtdanningSkalDuTa = tilleggsstonad.hvilketKursEllerAnnenFormForUtdanningSkalDuTa,
+        oppgiHvorMangeProsentDuStudererEllerGarPaKurs = tilleggsstonad.oppgiHvorMangeProsentDuStudererEllerGarPaKurs
+            ?: 0,
+        harDuEnFunksjonshemningSomGirDegStorreUtgifterTilLaeremidler = tilleggsstonad.harDuEnFunksjonshemningSomGirDegStorreUtgifterTilLaeremidler
```

Comment:
Se over alle properties og heller bruk `validateNotNull` i stedet for default verdier.